create_project_version.rb

```diff
@@ -0,0 +1,30 @@
+module CreateProjectVersion
+
+  def create_project_version
+    @driver.navigate.to 'http://demo.redmine.org'
+    @driver.find_element(:class, 'register').click
+
+    @wait.until {@driver.find_element(:id, 'user_login').displayed?}
+
+    @login = 'login'+rand(99999).to_s
+    @password = 'Qwerty11'
+    @driver.find_element(:id, 'user_login').send_keys(@login)
+    @driver.find_element(:id, 'user_password').send_keys(@password)
+    @driver.find_element(:id, 'user_password_confirmation').send_keys(@password)
+    @driver.find_element(:id, 'user_firstname').send_keys('user_firstname')
+    @driver.find_element(:id, 'user_lastname').send_keys('user_lastname')
+    @driver.find_element(:id, 'user_mail').send_keys(@login + '@q.com')
+    @driver.find_element(:name, 'commit').click
+    @driver.find_element(:css, ".projects").click
+    @driver.find_element(:css, ".icon.icon-add").click
+    @project_name = 'Project' + rand(9999).to_s
+    @driver.find_element(:css, "#project_name").send_keys(@project_name)
+    @driver.find_element(:xpath, ".//*[@id='new_project']/input[@name='commit']").click
+    @driver.find_element(:id, 'tab-versions').click
+    @driver.find_element(:link, 'New version').click
```

Comment:
The Redmine project is multilingual and if you'll try to run your tests on the machine where browser have other language as default a lot of actions based on elements' text will fail. Try to prevent it